int64 highScore
int64 roundsComplete
int64 losses
bool gameStart
int64 highScoreDiff
